README.md

```diff
@@ -12,60 +12,84 @@ promotion from universe to main.
-| 4. In Progress (any)                    | MIR team ack (and if needed Security Team ack) done, but now needs the Dependency/Seed change to happen to pull it into Main |
-| 5. Fix Committed (any)                  | All of the above done, waiting for an AA (Archive Admin) to promote the packages |
-| 6. Fix Released (any)                   | Case resolved by an AA |
-| 7. Incomplete (any)                     | Questions/Requests were raised for the bug reporter to resolve/clarify |
-| 8. Won\'t Fix (any)                     | Final Nack or given up by the bug reporter |
-| 9. Invalid (any)                        | No Feedback for a while when in incomplete status |
-
-```
-          Answered       +-7----------+     No feedback        +-9-------+
-      +------------------+ Incomplete +------------------------> Invalid |
-      |   old assign can | any        <--------------------+   | any     |
-      |   be kept as-is  +-----^------+                    |   +---------+
-      |                        |    question               |
-      |                        |                           |
-+-1---v-------+  Triage  +-2---+----+ Security needed  +-3-+------------------+
-|  New/Conf   +--------->+ New/Conf +----------------->+ New/Conf             |
-| unassigned  |          | assigned |                  | assigned to Security |
-+-------------+          +-+--+--+--+                  +-+--+--+--------------+
-                           |  |  |    +-8-----------+    |  |  |
-                           |  |  +----> Won't Fix   <----+  |  |
-                           |  |   Nack| any         | Nack  |  |
-                           |  |       +-------------+       |  |
-                           |  |                             |  |
-                           |  |       +-4-----------+       |  |
-                           |  +-------> In Progress <-------+  |
-                           |    Ack   | any         |  Ack     |
-                           |          +-------------+          |
-                           |             |Change that pulls    |
-                           |             |into Main            |
-+-6------------+           |         +-5-v-----------+         |
-| Fix Released |           +---------> Fix Committed <---------+
-| any          <---------------------+ any           |
-+--------------+  AA promotes        +---------------+
+## Process States
+
+This overview aims to clarify the meaning of MIR bug states, address previous 
+confusion, and indicate the next course of action on a MIR bug.
+
+```mermaid
+%% mermaid flowcharts documentation: https://mermaid.js.org/syntax/flowchart.html
+%%{ init: { 'flowchart': { 'curve': 'monotoneY' } } }%%
+flowchart TD 
+    subgraph New["New / Confirmed¹"]
```

Comment:
An important aspect of the old graph was that each valid state (anything not in the graph is invalid) has a tuple of Assignment+State. This is somewhat lost now. A few examples:

This is great, but currently I miss a bit of continuity.
At the top the state set for new/confirmed is a yellowish box.
Down there all the state set actions for "Won't Fix", "In Progress" are purple boxes.
But at the top purple boxes are not state changes, they are assignee changes.

Could we make all state-changes purple boxes, and make all assignment changes e.g. yellow hexagons or something like that. So that the shape/color directly indicates what is expected to happen at this point?